src/modules/id.ts

```diff
@@ -1,21 +1,20 @@
-const { customAlphabet } = require("nanoid");
+import { customAlphabet } from "nanoid";

```

Comment:
Will this work of I update nanoid to v4.0.0 which uses esm6?

Reply:
It should work as long as you have the 'nanoid' package installed and properly imported.